src/GenAPI/Microsoft.DotNet.GenAPI/IAssemblySymbolWriter.cs

```diff
@@ -0,0 +1,15 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Microsoft.CodeAnalysis;
+
+namespace Microsoft.DotNet.GenAPI
+{
+    /// <summary>
+    /// Interface incapsulates logic for processing symbol assemblies.
+    /// </summary>
+    internal interface IAssemblySymbolWriter
+    {
+        void WriteAssembly(IAssemblySymbol assemblySymbol);
```

Comment:
Please add triple slash xml comments for this method. (as it's public)

Reply:
this is done